5.x/crud-fields.md

```diff
@@ -483,6 +483,9 @@ Include an `<input type="hidden">` in the form.
 <a name="month"></a>
 ### month
 
+**IMPORTANTE NOTICE**: This field type is not supported by all browsers. [View compatibility table](https://caniuse.com/mdn-html_elements_input_type_month)
+See `#workaround` section below for ideas on how to solve this.
+
```

Comment:
```suggestion
Include an `<input type="month">` in the form.

**Important**: This input type is not supported by all browsers. [View compatibility table](https://caniuse.com/mdn-html_elements_input_type_month). See a workaround below.

```

Reply:
Done this because it feel odd to start the section with CAREFUL, THIS DOESN'T WORK.

Also, rephrased so it's more positive - it DOES work, in most browsers. Doesn't in a few.